composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "mnapoli/bref",
+    "name": "bref/bref",
```

Comment:
You can add:
```
"replace": {
        "mnapoli/bref": "*"
}
```
https://getcomposer.org/doc/04-schema.md#replace

Reply:
Thanks, I'm not certain we need this, especially since we are not sure what will happen. Bref is not used much as a library, I'm not sure the gain is worth the risk.

Reply:
You don't need to rename the package name. 

I would actually suggest to keep it the same.